docker-compose.yml

```diff
@@ -18,9 +18,14 @@ services:
     networks:
     - backend
     environment:
-      LOCAL_CONTRACTS: 'true'
+      LOCAL_CONTRACTS: "true"
```

Comment:
are we sure we need this as a string here instead of a boolean?

Reply:
This is the check in keeper-contracts: `"${LOCAL_CONTRACTS}" = "true"
I didn't write this, originally.

Javier @jcortejoso, can this be a boolean?

Reply:
Yeah, `docker-compose` does not accept booleans.

Reply:
interesting, good learning, thanks